server/server.js

```diff
@@ -269,10 +269,10 @@ This endpoint sends a put request to the Client Response API
 for modifying a single Client Response by its Response GUID,
```

Comment:
Please modify the comment on line 268 to say "patch" now.

Reply:
Done